config.json

```diff
@@ -12,7 +12,8 @@
   "require": {
     "moment": "moment",
     "_": "underscore",
-    "typogr": "typogr"
+    "typogr": "typogr",
+    "lo": "lodash"
```

Comment:
You can remove lodash. I was using some of its functions in lifemap. Now
i've moved all its codebase to its own repo.

On Sun, Nov 29, 2015 at 8:21 AM, Jeremy Scheff <notifications@github.com>
wrote:

> In config.json
> <https://github.com/Data4America/simple-static-stories/pull/6#discussion_r46087101>
> :
>
> > @@ -12,7 +12,8 @@
> >    "require": {
> >      "moment": "moment",
> >      "_": "underscore",
> > -    "typogr": "typogr"
> > +    "typogr": "typogr",
> > +    "lo": "lodash"
>
> We really need underscore and lodash?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Data4America/simple-static-stories/pull/6/files#r46087101>
> .
>